Formula/httpd.rb

```diff
@@ -0,0 +1,132 @@
+class Httpd < Formula
+  desc "HTTP server"
+  homepage "https://httpd.apache.org/"
+  url "https://archive.apache.org/dist/httpd/httpd-2.4.27.tar.bz2"
+  sha256 "71fcc128238a690515bd8174d5330a5309161ef314a326ae45c7c15ed139c13a"
+
+  bottle do
```

Comment:
For other migrations, we've rebuilt bottles, IIRC

Reply:
Yeh, will do when we pull.